packages/react-composites/src/composites/CallWithChatComposite/ChatButtonWithUnreadMessagesBadge.tsx

```diff
@@ -27,20 +27,25 @@ export interface ChatButtonWithUnreadMessagesBadgeProps extends ControlBarButton
 const validNewChatMessage = (message): boolean =>
   !!message.senderDisplayName && (message.type === 'text' || message.type === 'html');
 
+const filledIcon = <Chat20Filled key={'chatIconKey'} primaryFill="currentColor" />;
```

Comment:
why was this logic moved out of the chat button?

Reply:
It is needed here because the icon is chosen in two places -- once in `ChatButton` and once in the icon used in the `Stack` in `notificationOnIcon`.

Moved it here to deduplicate the logic about which icon to use.

Reply:
ahh okay makes sense!